### ISSUE TITLE
Remove old Sentry init and dependencies

### ISSUE THREAD

**Issue by BlishHUD** [blish-hud/Blish-HUD#138]
**Greaka at 10/30/2019 8:12:47 AM +00:00**
> it would be nice to get rid of all the sentry stuff at some point
> we can still build sentry releases and i missclick all the time 🙈 
> i dont know what has to be purged though :/

REF: https://discordapp.com/channels/531175899588984842/536970543736291346/639013821813620737

**Comment by dlamkins** [comment]
Some additional details about the current state of Sentry:

While very helpful early on, there were some concerns from others about privacy regarding what kind of information was being collected as part of the logging process.  I did my best to be as transparent as possible and to strip as much identifying information as possible from the collection process, but I understand that any kind of logging can make folks wary - and that's not how I want people to feel when using Blish HUD.  Additionally, after the work was done to improve Blish HUD's local logging with NLog, Sentry no longer was the best source of information when troubleshooting issues as the local logs contained much more valuable data regarding what Blish HUD was doing at the time of the crash.  Finally, these logs are sent to us from users directly who we can then discuss and troubleshoot solutions with directly which provides a far better troubleshooting experience compared to wrestling with anonymous error messages with limited context.

For these reasons, I mentioned that we would no longer be putting out builds that had Sentry enabled on 8/26/2019.  On 9/29/2019, I was reminded that previous dev releases in our #blish_releases Discord channel still included the Sentry builds for download, so those Sentry builds were deleted from the channel.

At this time the code and dependencies for Sentry are still in the application.  A special flag must be passed when compiling Blish HUD to enable that logging which is how we were able to easily automate the generation of two version of the application (one with Sentry and one without) without needing to maintain separate codebases.  Despite being essentially inaccessible by the application, the dependency still remains in every build put out at this time.  Since it is no longer used, it should be fully removed.
